02-oop-inheritance/dist/script.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
Please add `dist` to the `.gitignore` at the root of the repository. This is not intended to be commited